src/v2/view-builder/views/consent/EnduserEmailConsentView.js

```diff
@@ -0,0 +1,91 @@
+import { loc } from 'okta';
+import { BaseForm, BaseHeader } from '../../internals';
+import ConsentViewForm from './ConsentViewForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import HeaderBeacon from '../../components/HeaderBeacon';
+import { getIconClassNameForBeacon } from '../../utils/AuthenticatorUtil';
+import { AUTHENTICATOR_KEY } from '../../../ion/RemediationConstants';
+
+export default BaseAuthenticatorView.extend({
+  className: 'enduser-email-consent',
+
+  Header: BaseHeader.extend({
+    HeaderBeacon: HeaderBeacon.extend({
+      getBeaconClassName: function() {
+        return getIconClassNameForBeacon(AUTHENTICATOR_KEY.EMAIL);
+      },
+    }),
+  }),
```

Comment:
This is needed because - 

1. This is a fresh SIW that is bootstrapped. There's no `authenticatorKey` set in `appState`. Also, `Beacon` is necessary here because the `TerminalView`displays Beacons based on the `i18n` key received in remediation.

2. One of the terminal states has been reused in this flow - `idx.return.to.original.tab` which already shows Email Beacon (`playground/mocks/data/idp/idx/terminal-return-email-consent.json`). So, if I change that flow to not show Email Beacon, I might be breaking other flows. Hence this was needed for consistency in this flow although UX preferred not to have Beacon for this.

Reply:
@vincentvoong-okta - so TLDR; we need to have the email beacon in this flow because of shared terminal flows.